src/Traits/ReflectionClassLikeTrait.php

```diff
@@ -519,7 +518,8 @@ function (array &$result, \ReflectionClass $instance, $isParent) {
     /**
      * {@inheritdoc}
      */
-    public function getProperty($name)
+    #[\ReturnTypeWillChange]
```

Comment:
@lisachenko  this is needed due to original Reflection return `ReflectionProperty`, while it can return `ReflectionProperty|false`

Reply:
Yes, confirmed, thank you